.github/workflows/ci.yml

```diff
@@ -0,0 +1,126 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - "release/**"
+env:
+  APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+  APPLITOOLS_BATCH_ID: ${{ github.event.pull_request.head.sha }}
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      deploy: ${{ steps.filter.outputs.deploy }}
+      vrt: ${{ steps.filter.outputs.vrt }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            deploy:
+              - './packages/odyssey-react/**'
+              - './packages/odyssey-icons/src/**'
```

Comment:
why the icons package? I don't think these are exposed anywhere outside of odyssey-react yea? And they only make their way into the package via an explicit build step run from `odyssey-react` ?